.github/workflows/build.yml

```diff
@@ -0,0 +1,65 @@
+name: Build VS Code extension
+
+on:
+  push:
+  pull_request:
+  release:
+    types: [published]
```

Comment:
Isn't a limitation of the default github actions token (`GH_TOKEN`), that it can't trigger other actions? Won't this be an issue here since it is this token that creates the release?

Btw in `bazel-lsp` I am using a [slightly different workflow](https://github.com/cameron-martin/bazel-lsp/blob/master/.github/workflows/build-release.yml) that only runs the release job after building. This works around this issue and prevents building again for the release. The downside is that you prevent being able to build differently for releases, but I wonder if this is a good idea anyway.

Reply:
> The downside is that you prevent being able to build differently for releases, but I wonder if this is a good idea anyway.

My mid-term vision for the automation here would be:
* automatically upload releases also to the Marketplace by calling `vsce`
* build every `master` commit and publish it the Marketplace as a pre-release

but that seems to also be possible with your way to structure the workflow

Reply:
Publishing pre releases to the marketplace would be very neat. Hopefully we get control over that when moving to bazel-contrib.